src/components/CardText/index.tsx

```diff
@@ -32,8 +32,6 @@ export const CardText = styled.p<CardTextProps>(
   font-weight: ${props.fontWeight === 'light' ? 400 : 500};
   line-height: ${props.fontSize === 'title' ? '19px' : '17px'};
   height: ${props.fontSize === 'title' ? '19px' : '17px'};
```

Comment:
Can't we use CSS media querys for this

https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/

Anyway, is this the standard react way? Because I think  CSS and logic should be separated, but I may be just oldschool.